src/db.py

```diff
@@ -48,8 +48,14 @@ class Receipt(Base):
 class DBDriver:
 
     def __init__(self):
-        self._db_url = os.environ.get("DATABASE_URL")
-        self._engine = create_engine(self._db_url)
+        self.DB_NAME = os.environ.get("DB_NAME")
```

Comment:
1. Not sure that all these new properties will be used later. Let's keep them as variables.
2. In general, they are custom properties and should start with _ (e.g. self._engine).